tests/cat/health.yaml

```diff
@@ -82,3 +82,16 @@ chapters:
           status: yellow
           node.data: '1'
           discovered_cluster_manager: 'true'
+  - synopsis: Cat in different formats (format=smile).
+    method: GET
+    path: /_cat/health
+    parameters:
+      format: smile
+    response:
+      status: 200
+      content_type: application/yaml
```

Comment:
Are we not actually validating `content_type` against the response content-type header?

Reply:
Looks like it, https://github.com/opensearch-project/opensearch-api-specification/issues/392.